### PR TITLE
update specs

### PR DESCRIPTION
**Pull request by apodpalok**

### CODE REVIEW

File: spec/controllers/trips_spec.rb [hunk 1]
```diff
@@ -111,10 +109,12 @@
   describe 'DELETE#destroy' do
     before { login_user }
 
-    trip = FactoryGirl.create(:trip)
+    let(:trip) { FactoryGirl.create(:trip) }
```
Comment (abscondite) [inline spec/controllers/trips_spec.rb:112]: You can use `#let!` instead:
`let!(:trip) { FactoryGirl.create(:trip) }`

File: spec/controllers/trips_spec.rb [hunk 2]
```diff
@@ -111,10 +109,12 @@
...
+
     subject { delete :destroy, params: { id: trip.id } }
 
     it 'delete trip' do
+      trip
```
Comment (abscondite) [inline spec/controllers/trips_spec.rb:117]: And this will be unnecessary

**Comment by abscondite** [comment]
Also you can use just `create(...)` instead of `FactoryGirl.create(...)`

**Comment by apodpalok** [comment]
Thanks! It looks better!

**Pull request merged**